client/my-sites/woocommerce/dashboard/required-plugins-install-view.jsx

```diff
@@ -475,6 +514,38 @@ class RequiredPluginsInstallView extends Component {
 		);
 	}
 
+	renderContactSupportForTimeout() {
+		const { translate } = this.props;
+
+		const subtitle = [
+			<p key="line-1">
+				{ translate( 'Sorry, we had a problem setting up your store.', {
+					components: { b: <strong /> },
```

Comment:
We don't need this `components`.

Reply:
Thanks for catching that! Resolved in f57ebe7